include/globaldata.h

```diff
@@ -4,5 +4,6 @@
 // This file gets included by each source file, just like in Chunsoft's original source, so we can have this string easily just like they did.
 
 // Later, we can move this into "global.h" before the #ifdef guard. But we can only do this if "global.h" isn't included outside of .c files...
+#include "gba/defines.h"
```

Comment:
Whoops, didn't mean to commit this, this was making coccinelle spew an annoying amount of errors on that ALIGNED(4) attribute if this was missing